src/api/app/models/user.rb

```diff
@@ -176,59 +176,14 @@ def self.create_user_with_fake_pw!(attributes = {})
     create!(attributes.merge(password: SecureRandom.base64(48)))
   end
 
-  def self.create_ldap_user(attributes = {})
-    user = create_user_with_fake_pw!(attributes.merge(state: default_user_state, adminnote: 'User created via LDAP'))
-
-    return user if user.errors.empty?
-
-    logger.info("Cannot create ldap userid: '#{login}' on OBS. Full log: #{user.errors.full_messages.to_sentence}")
-    nil
```

Comment:
These lines were not moved to `find_with_credentials` in `src/api/app/models/user_ldap_strategy.rb`. But I guess that exception handling when creating an user fails is enough.